packages/notification/types.ts

```diff
@@ -4,6 +4,7 @@ export type Renderable = JSX.Element | string | null
 
 export type ValueFunction<TValue, TArg> = (arg: TArg) => TValue
 export type ValueOrFunction<TValue, TArg> = TValue | ValueFunction<TValue, TArg>
+export type NotificationAsType = 'toast' | 'dialog'
```

Comment:
nit: I think this type name would be better as `NotificationType`.

Reply:
Updated https://github.com/hashicorp/react-components/pull/656/commits/a7e73d52f83d5a43f7989df4d832110033e15e75